### DISCUSSION TITLE
any fullblown apps on github using Mojo?

### DISCUSSION THREAD

**Discussion by ralyodio** [mojolicious/mojo#1956] [Q&A]
I'm starting to use it on a side project and am curious how they organize things like:

- db migrations
- authentication (jwt)
- client side javascript (ie: web components)
- .env.* files

I'm starting one here: https://github.com/profullstack/iptvcafe if anyone wants to help.

**Comment by kraih** [comment]
* https://docs.mojolicious.org/Mojolicious/Guides/Growing
* https://github.com/mojolicious/mojo-pg/tree/main/examples/blog
* https://github.com/openSUSE/qem-dashboard
* https://github.com/openSUSE/cavil

**Comment by jhthorsen** [comment]
https://github.com/convos-chat/convos

**Comment by flash548** [comment]
Not full blown, a WIP, but does authentication.  Started as a Lite app, but I'm slowing growing it and breaking it out into controllers/services/etc.   Uses JWT (or produces them rather), Preact (so not web-components), and Mojo::Pg/ Mojo::SQLite. 

https://github.com/flash548/mojo-gateway

**Comment by s1037989** [comment]
I am very interested in this project!  Thanks for sharing!  This seems like
a project I'd be interested in contributing to if there's anything that
could use help.

**Reply by flash548** [reply]
Thanks, yeah I need to make a list of features and tasks needing done.  Right now I'm just getting it to a point where it's stable with a full test suite.  I'll post of of those in the near future along with contrib guidelines.  But pull requests/ideas/etc are always welcome!

**Comment by ralyodio** [comment]
Here's what I was looking for: `npm create @mojojs/full-app`
